custom/icds_reports/sqldata/base_operationalization.py

```diff
@@ -35,3 +39,42 @@ def rows(self):
                     0
                 ]
             ]
+
+
+class BaseOperationalizationBeta(ICDSMixin):
+
+    title = 'c. Status of operationalization of AWCs'
+    slug = 'operationalization'
+
+    @property
+    def headers(self):
+        return DataTablesHeader(
+            DataTablesColumn('', sortable=False),
+            DataTablesColumn('Sanctioned', sortable=False),
+            DataTablesColumn('Functioning', sortable=False),
+            DataTablesColumn('Reporting', sortable=False)
+        )
+
+    @property
+    def rows(self):
+
+        if self.config['location_id']:
+            filters = get_location_filter(self.config['location_id'], self.config['domain'])
+
+            if filters.get('aggregation_level') > 1:
+                filters['aggregation_level'] -= 1
+
+            filters['month'] = date(self.config['year'], self.config['month'], 1)
+            awc_data = AggAwc.objects.filter(**filters).values(
+                'num_awcs',
+                'num_launched_awcs',
```

Comment:
the old version of this appears to define the second two indicators identically. why does this distinguish?

Reply:
While going through the MPR report, keeping the two same was not making sense and not adding any value. So discussed with the team and we decided to make this change.